packages/solid/src/index.ts

```diff
@@ -28,7 +28,8 @@ export {
   runWithOwner,
   tryCatch,
   untrack,
-  unwrap
+  unwrap,
+  deep,
```

Comment:
I was also looking to export the `path` helper but I don't think that's implemented yet